src/App.js

```diff
@@ -1,16 +1,54 @@
-import React from 'react';
+import React from "react";
+import Search from "./components/Search";
+import MovieResults from "./components/MovieResults";
+import Info from "./components/Info";
 
 class App extends React.Component {
-  constructor(){
-    super();
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      movies: [],
+      moreInfo: []
+    };
+
+    this.getMovies = this.getMovies.bind(this);
+    this.getMoreInfo = this.getMoreInfo.bind(this);
+    this.hideInfo = this.hideInfo.bind(this);
+  }
+
+  getMovies(result) {
+    this.setState({
+      movies: result
+    });
+  }
+
+  getMoreInfo(result) {
+    this.setState({
+      moreInfo: result
+    });
+  }
+
+  hideInfo() {
+    this.setState({
+      moreInfo: ""
+    });
   }
 
-  render(){
+  render() {
     return (
-      <div>
-        React cinema app
+      <div className="background">
+        <Search getMovies={this.getMovies} hideInfo={this.hideInfo} />
+        <MovieResults
+          movies={this.state.movies}
+          getMoreInfo={this.getMoreInfo}
+        />
+        {this.state.moreInfo === "" ? null : (
```

Comment:
this might not work since `moreInfo` is initialised to empty array. You could also simplify it to take advantage of empty string being falsy by writing it as `!this.state.moreInfo ? null : <Info moreInfo={this.state.moreInfo} />`